torchdynamo/codegen.py

```diff
@@ -110,6 +130,28 @@ def __call__(self, value, allow_cache=True):
                 self._create_load_const(graph_outputs[graph_outputs_key].index)
             )
             output.append(create_instruction("BINARY_SUBSCR"))
+
+            if isinstance(value, UnspecializedNumpyVariable):
+                unspec_var = self.tx.output.new_var("unspec")
+                output.extend(
+                    [
+                        self.create_load_attr("item"),
+                        create_instruction("CALL_FUNCTION", 0),
+                        self.create_store(unspec_var),
+                        self.create_load_global("type", add=True),
+                        self.create_load_const(value.raw_value),
```

Comment:
since `type(value.raw_value)` is a constant, perhaps just compute it at compile time?